src/main/java/eventprocessing/DataReader.java

```diff
@@ -0,0 +1,21 @@
+package eventprocessing;
+
+import com.amazonaws.services.s3.model.S3ObjectInputStream;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+
+public class DataReader {
+    public void saveToFile(S3ObjectInputStream amazonS3Stream) throws IOException {
+        FileOutputStream fos = new FileOutputStream(new File("locations.json"));
```

Comment:
"locations.json" is hardcoded in a couple of different places. It's also easy to mix up with the S3 key. What about moving into the config?